tasks/run.py

```diff
@@ -43,21 +43,26 @@ def shell(ctx):
   with ctx.cd(ROOT_REPO_DIR):
     ctx.run(f'flask shell', env=FLASK_ENV, pty=True)
 
-@task()
-def gunicorn(ctx):
+@task(
+  help={
+    'force_prod': 'Set the IN_PRODUCTION variable to emulate production',
+  }
+)
+def gunicorn(ctx, force_prod=False):
```

Comment:
Should I be using gunicorn instead of just inv.run? I'm confused about this now.

Reply:
you're not supposed to deploy the built-in flask web server in production. in production we run `gunicorn`, and you can run it locally, too, but you'll probably be happier with `inv run` and it's interactive debugger and automatic code reloading